common/exception/ErrorMessage.java

```diff
@@ -86,6 +88,7 @@
 
     VALIDATION_CASTING("The type [%s] of role player [%s] is not allowed to play Role [%s] \n"),
     VALIDATION_ROLE_TYPE_MISSING_RELATION_TYPE("Role [%s] does not have a relates connection to any Relation Type. \n"),
+    VALIDATION_ROLE_TYPE_MULTIPLE_NON_HIERARCHY_RELATION_TYPE("Role [%s] has multiple relates connection to Relation Types that are not in a hierarchy. \n"),
```

Comment:
What does this error message mean?

Reply:
renamed to `VALIDATION_ROLE_TYPE_TOO_MANY_NON_RELATED_RELATION_TYPE`